roles/openshift_examples/files/examples/x86_64/image-streams/image-streams-centos7.json

```diff
@@ -909,6 +909,26 @@
             "referencePolicy": {
               "type": "Local"
             }
+          },
+          {
+            "annotations": {
+              "description": "Build and run PHP 7.2 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-php-container/blob/master/7.2/README.md.",
+              "iconClass": "icon-php",
+              "openshift.io/display-name": "PHP 7.2",
+              "openshift.io/provider-display-name": "Red Hat, Inc.",
+              "sampleRepo": "https://github.com/openshift/cakephp-ex.git",
+              "supports": "php:7.2,php",
+              "tags": "builder,php",
+              "version": "7.2"
+            },
+            "from": {
+              "kind": "DockerImage",
+              "name": "docker.io/centos/php-72-centos7:latest"
```

Comment:
we don't have this on quay or registry.redhat.io?  i'd prefer to avoid using docker.io if we can due to the rate limiting.

Reply:
yeah `quay.io/centos/php-72-centos7:latest` does not exist @bparees and I had no luck searching for a viable alternative on quay.io (though I'm not overlay impressed with its search function ... tends to provide too many false positives IMO)

if you have the cycles to search and do better than me, please let me know

Reply:
my thought was that it had to by centos vs. rhel ... and even then had to be registry.access.redhat.com 

but provide me official overrides on those points as you deem fit

Reply:
you're probably correct but we should talk to the SCL team about getting their stuff published on quay instead.  (I have no idea what is building+pushing those centos images these days)

i'll lgtm for now and hopefully our usage stays low enough that it doesn't break CI